index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="main.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <main id="page-maincontainer">
+      <div class="overlay"></div>
+      <div class="form-wrapper">
+        <h1 id="title">Survey Form</h1>
```

Comment:
- [ ] It could be better to prefer using the class selector over the ID selector because sometimes you may want to use the same styling for other elements, using IDs it can't be possible because they are supposed to be unique, but with class selector, you can do this. learn more about the difference between ID and class selectors [here](https://css-tricks.com/the-difference-between-id-and-class/).

Reply:
- [ ] It will be nice to explain why the class selector is better in that case than an Id selector, thus you will show interest in the learning process of the student. Please be more polite in your review, it is not a good practice to be hard on the student, kindness reinforces self-confidence.